### PR TITLE
Mailer

### PR DESCRIPTION
**Pull request by brokenheart1712**

### CODE REVIEW

File: app/mailers/application_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: "Framgia Test App"
+  layout 'mailer'
```
Comment (NguyenTanDuc) [inline app/mailers/application_mailer.rb]: ' => "

File: app/views/user_mailer/result_notification.html.erb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<h1>Framgia Test App</h1>
```
Comment (NguyenTanDuc) [inline app/views/user_mailer/result_notification.html.erb]: I18n

File: app/views/user_mailer/result_notification.text.erb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Framgia Test App
```
Comment (NguyenTanDuc) [inline app/views/user_mailer/result_notification.text.erb]: I18n

File: app/views/user_mailer/exam_notification.text.erb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+Framgia Test App
```
Comment (NguyenTanDuc) [inline app/views/user_mailer/exam_notification.text.erb]: I18n

File: app/views/user_mailer/exam_notification.html.erb [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<h1>Framgia Test App</h1>
```
Comment (NguyenTanDuc) [inline app/views/user_mailer/exam_notification.html.erb]: I18n

**Comment by NguyenTanDuc** [comment]
OK

File: app/mailers/application_mailer.rb [hunk 6]
```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: t :fts
```
Comment (dieunb) [inline app/mailers/application_mailer.rb]: Using gem [setting](https://github.com/railsconfig/rails_config) to store your email

File: app/views/user_mailer/exam_notification.html.erb [hunk 7]
```diff
@@ -0,0 +1,4 @@
+<h1><%= t :fts %></h1>
```
Comment (dieunb) [inline app/views/user_mailer/exam_notification.html.erb]: Email content don't use I18n

Comment (brokenheart1712) [inline app/views/user_mailer/exam_notification.html.erb]: Sure.

File: app/mailers/application_mailer.rb [hunk 8]
```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: Settings.default_mailer_name
```
Comment (dieunb) [inline app/mailers/application_mailer.rb]: Email, why is name?

File: app/views/user_mailer/exam_notification.html.erb [hunk 9]
```diff
@@ -0,0 +1,4 @@
+<h1>Framgia Test App</h1>
+<p>Dear <%= @user.name %>,</p>
+<p>Click On This Link To Start The Exam: </p>
+<%= link_to "Start Exam!", "#" %>
```
Comment (dieunb) [inline app/views/user_mailer/exam_notification.html.erb]: href is null???

File: app/views/user_mailer/exam_notification.text.erb [hunk 10]
```diff
@@ -0,0 +1,4 @@
...
+Dear <%= @user.name %>,
+Click On This Link To Start The Exam:
+<%= link_to "Start Exam!", "#" %>
```
Comment (dieunb) [inline app/views/user_mailer/exam_notification.text.erb]: href is null???

File: app/views/user_mailer/result_notification.html.erb [hunk 11]
```diff
@@ -0,0 +1,4 @@
...
+<p>Dear <%= @user.name %>,</p>
+<p>Click On This Link To View The Result:</p>
+<%= link_to "View Result!", "#" %>
```
Comment (dieunb) [inline app/views/user_mailer/result_notification.html.erb]: href is null???

File: app/views/user_mailer/result_notification.text.erb [hunk 12]
```diff
@@ -0,0 +1,4 @@
+<h1>Framgia Test App</h1>
+<p>Dear <%= @user.name %>,</p>
+<p>Click On This Link To View The Result: </p>
+<%= link_to "View Result!", "#" %>
```
Comment (dieunb) [inline app/views/user_mailer/result_notification.text.erb]: href is null???

**Pull request merged**